src/clr-angular/data/datagrid/datagrid-column.ts

```diff
@@ -228,6 +240,10 @@ export class ClrDatagridColumn<T = any> extends DatagridFilterRegistrar<T, Datag
     }
   }
 
+  public get ariaButtonLabel() {
```

Comment:
Our pattern has really been to just make the commonStrings public (like you have) and then reference the field directly without a getter like this.

Reply:
This is remainder of the 1st attempt to supply text like "Sort on Name". Not needed now.

Reply:
This was because I try to get the name of the `columnTitle` and `concat` them. If we stay with this solution it's no need for this type `set/get` function. 

Will leave this discussion open, to remind me to rework it later on